python.html.markdown

```diff
@@ -568,7 +568,7 @@ dir(math)
 ## 7. Advanced
 ####################################################
 
-# Generators help you make lazy code
+# Generators help you make efficient code
```

Comment:
"Lazy" was the precise description for generators. It explains that the values are lazily computed. "Efficient," on the other hand, is much more general, and therefore less descriptive.